vividus-facade-jira/src/main/java/org/vividus/jira/JiraConfigurationProvider.java

```diff
@@ -0,0 +1,82 @@
+package org.vividus.jira;
```

Comment:
```suggestion
package org.vividus.jira.config;
```
? and move to this package everything related to the configuration?